packages/manfred2html/src/engine/cv-jedi-minimalism/html-parts/html-document-start/html-document-start.part.ts

```diff
@@ -0,0 +1,5 @@
+import ejs from 'ejs';
+import { HexColor } from '@/model';
```

Comment:
This CV need a new HexColor enum with its values.

Reply:
Yes, but not only this, you need a new mapper and use it. Check if the colors are corect when you select a color know.